janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -199,10 +190,12 @@
     private final boolean useExternalMappings;
     private final Map<String,Object> indexSetting;
     private final long createSleep;
+    private final boolean useAllField;
     private final boolean useMultitypeIndex;
 
     public ElasticSearchIndex(Configuration config) throws BackendException {
         indexName = config.get(INDEX_NAME);
+        useAllField = config.get(USE_ALL_FIELD);
         useExternalMappings = config.get(USE_EXTERNAL_MAPPINGS);
```

Comment:
As you add an _all_ field, I think for _query_string_, you should set the _default_field_ in the query_string to _all_ for ES >=2 because copy_to was add in ES 2